src/components/Description/DescriptionLLM/index.tsx

```diff
@@ -1,99 +1,101 @@
-import React, { PureComponent } from 'react';
+import React from 'react';
 
 import cssBinder from 'styles/cssBinder';
 import globalStyles from 'styles/interpro-vf.css';
 import localStyles from './style.css';
 import fonts from 'EBI-Icon-fonts/fonts.css';
 import Link from 'components/generic/Link';
 
-import Description from '..';
 import config from 'config';
 
 const css = cssBinder(globalStyles, localStyles, fonts);
 
 type Props = {
   accession: string;
-  text: string;
 };
 
-class DescriptionLLM extends PureComponent<Props> {
-  render() {
-    const accession = this.props.accession || '';
-    const text = this.props.text || '';
-    if (accession.length === 0 || text.length === 0) return null;
+const DescriptionLLM = ({ accession }: Props) => {
+  if ((accession || '').length === 0) return null;
 
-    return (
-      <>
-        <div className={css('row')}>
-          <div className={css('columns', 'large-12')}>
-            <div
-              className={css('callout', 'warning')}
-              style={{
-                display: 'flex',
-                alignItems: 'center',
-              }}
+  return (
+    <div className={css('vf-stack', 'vf-stack--400')}>
+      <div
+        className={css('callout', 'warning')}
+        style={{
+          display: 'flex',
+          alignItems: 'center',
+        }}
+      >
+        <span
+          style={{
+            fontSize: '2em',
+            color: '#664d03',
+            paddingRight: '1rem',
+          }}
+          className={css(
+            'small',
+            'icon',
+            'icon-common',
+            'icon-exclamation-triangle',
+          )}
+        />{' '}
+        <p>
+          Some paragraphs of this description have been automatically generated
+          using{' '}
+          <Link href="https://openai.com/research/gpt-4" target="_blank">
+            GPT-4
+          </Link>
+          , an AI language model, and is based on data extracted from{' '}
+          <Link href="https://www.uniprot.org" target="_blank">
+            UniProtKB/Swiss-Prot
+          </Link>
+          .{' '}
+          <b>
+            Check the icon{' '}
+            <span
+              className={css('small', 'icon', 'icon-common', 'icon-magic')}
+            />{' '}
+            at the top of each paragraph to identify generated paragraphs. An
+            additional{' '}
+            <span
+              className={css('small', 'icon', 'icon-common', 'icon-check')}
+            />{' '}
+            icon indicates that a curator has approved the generated text.
+          </b>{' '}
```

Comment:
Using text instead of icons (see other suggestion) so we don't need additional text in the callout.

```suggestion
```